meta-resin-common/recipes-support/resin-mounts/resin-mounts/mnt-conf-systemd.automount

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Automount systemd presistent state
+
+[Automount]
```

Comment:
I believe that this mount unit should be an automount to make it more managable. Since a couple of systemd services that use that directory depend on it, and more may appear in the future, it's simpler to make it an automount and don't think about dependencies. I don't think this reasoning applies to the other mounts.